src/components/molecules/ScheduleVenueDescription/ScheduleVenueDescription.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { FC, useMemo } from "react";
+
+import { useConnectRelatedVenues } from "hooks/useConnectRelatedVenues";
+import { useVenueId } from "hooks/useVenueId";
+
+import "./ScheduleVenueDescription.scss";
+
+export const ScheduleVenueDescription: FC = () => {
+  const venueId = useVenueId();
+
+  const { parentVenue, currentVenue } = useConnectRelatedVenues({
+    venueId,
+  });
+
+  // @debt: ideally this would find the top most parent of parents and use those details
+  const scheduleVenue = useMemo(
+    () => (parentVenue ? parentVenue : currentVenue),
+    [parentVenue, currentVenue]
+  );
```

Comment:
This will be changed, when https://github.com/sparkletown/sparkle/pull/1368 is merged